source/nodejs/adaptivecards-site/package.json

```diff
@@ -4,12 +4,13 @@
 	"private": true,
 	"scripts": {
 		"clean": "hexo clean",
-		"build": "hexo generate",
+		"build": "hexo generate && npx workbox-cli injectManifest workbox-config.json",
 		"release-deps": "npx lerna run --scope @microsoft/adaptivecards-site --include-dependencies release --stream",
 		"generate-md-ac": "node node_modules/marked-schema/bin/marked-schema ../../../schemas/adaptive-card.json ./schema-explorer-toc.yml -r AdaptiveCard",
 		"generate-md-hc": "node node_modules/marked-schema/bin/marked-schema ../../../schemas/host-config.json ./schema-hostconfig-toc.yml -r AdaptiveCardConfig",
-		"release": "hexo clean && hexo generate",
-		"start": "hexo server"
+		"release": "hexo clean && hexo generate && npx workbox-cli injectManifest workbox-config.json",
+		"start": "hexo server",
+		"generate-sw": "npx workbox-cli injectManifest workbox-config.json"
```

Comment:
use this task in `build` and `release` to avoid code repetition :)

Reply:
Good call out! Just updated these